### PR TITLE
chore: enable `noPropertyAccessFromIndexSignature` typescript rule

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1382/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/core-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/lists-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1382/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1382/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1382/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1382/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1382/playground/)

File: libs/components/forms/src/lib/modules/input-box/input-box.component.ts [hunk 1]
```diff
@@ -20,10 +20,12 @@ import { SkyInputBoxHostService } from './input-box-host.service';
...
- * To display an error message under the input, include a `sky-status-indicator` component with a `sky-control-error` class in the `sky-input-box` element, and use `*ngIf` to determine whether to display the error message based on the input value.
+ * Creates a wrapper to provide styling for SKY UX input components and  `input`, `textarea`, `select`, and `label` elements.
+ * To render the component correctly, include the `sky-form-control` class on the input element and the `sky-control-label`
+ * class on the `label` element. To display a help button beside the label, include a `sky-help-inline` component with a
+ * `sky-control-help` class in the `sky-input-box` element. To display a validation error message under the input, include a
+ * `sky-status-indicator` component with a `sky-control-error` class in the `sky-input-box` element. Use `*ngIf` to determine
+ * whether to display the error message based on the input value.
```
Comment (johnhwhite) [inline libs/components/forms/src/lib/modules/input-box/input-box.component.ts]: Was this change intended?

File: libs/components/forms/src/lib/modules/input-box/input-box.component.ts [hunk 2]
```diff
@@ -45,8 +47,8 @@ export class SkyInputBoxComponent implements OnInit {
 
   /**
    * Whether to visually highlight the input box as disabled. To disable the input box's
-   * `input` element, use the HTML `disabled` attribute or the Angular `FormControl.disabled`
-   * property. You must set both properties to disable an `input` element and visually indicate
+   * input element, use the HTML `disabled` attribute or the Angular `FormControl.disabled`
+   * property. You must set both properties to disable an input element and visually indicate
```
Comment (johnhwhite) [inline libs/components/forms/src/lib/modules/input-box/input-box.component.ts]: Was this change intended?

**Review by johnhwhite — APPROVED** [review]
